src/FileDownloadTrait.php

```diff
@@ -72,7 +71,7 @@ public function fileDownloadFrom($url) {
 
     /** @var Behat\Mink\Driver\BrowserKitDriver $driver */
     $driver = $this->getSession()->getDriver();
-    if ($driver instanceof Selenium2Driver) {
+    if (method_exists($driver, 'getAllCookies')) {
```

Comment:
Would it be better to check that the driver is instanceof of  `DriverInterface`? Since that is where getAllCookies comes from?

Reply:
Agree, will update PR. Have added an issue #123 to better explain my thinking here.

Reply:
Removed this change entirely for FileDownloadTrait - it looks like this is specific to the Selenium2 driver, so I won't propose changing it. Might benefit from adding comments to make it clear why this needs special casing for Selenium2 but I don't have that information.